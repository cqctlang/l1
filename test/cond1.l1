x = (1==1?12:11);
printf("%d\n", x);
x = (1==0?12:11);
printf("%d\n", x);
