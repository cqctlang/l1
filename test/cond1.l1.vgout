entry: var params, 1 locals, 0 temps
12
11
