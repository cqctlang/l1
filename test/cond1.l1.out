entry: 0 params, 0 locals, 0 temps
12
11
